src/frontend/qt_sdl/Input.cpp

```diff
@@ -36,6 +37,8 @@ u32 HotkeyPress, HotkeyRelease;
 
 u32 InputMask;
 
+bool isFrameDone = false;
```

Comment:
This should be Pascal-cased (though as mentioned in another comment I don't know if it's needed at all).